app/factories/flood_risk_engine/steps/form_object_factory.rb

```diff
@@ -38,11 +38,16 @@ def form_object_class_map
         private
 
         def setup_form_object(step)
-          "FloodRiskEngine::Steps::#{step.to_s.classify}Form".constantize
```

Comment:
I think `"#{step.to_s.classify}Form".constantize` would work as we already in the `FloodRiskEngine::Steps` name space